specs/test_formats/operations/README.md

```diff
@@ -24,7 +43,7 @@ Operations:
 |-------------------------|----------------------|----------------------|--------------------------------------------------------|
 | `attestation`           | `Attestation`        | `attestation`        | `process_attestation(state, attestation)`              |
 | `attester_slashing`     | `AttesterSlashing`   | `attester_slashing`  | `process_attester_slashing(state, attester_slashing)`  |
-| `block_header`          | `Block`              | `block`              | `process_block_header(state, block)`                   |
+| `block_header`          | `Block`              | **`block**           | `process_block_header(state, block)`                   |
```

Comment:
```suggestion
| `block_header`          | `Block`              | `block`              | `process_block_header(state, block)`                   |
```

Reply:
hmm, missed a backtick there. But bolding is intentional, as it highlights that this one item is different than the others.

Reply:
```suggestion
| `block_header`          | `Block`              | **`block`**           | `process_block_header(state, block)`                   |
```